src/components/document/FamilyDocument.tsx

```diff
@@ -71,11 +76,11 @@ export const FamilyDocument = ({ document, matches, status }: TProps) => {
           {renderContentType(content_type)}
           {!!language && (
             <span>
-              {language.toUpperCase()}
+              {getLanguage(language, languages)}
```

Comment:
This is the second place we create a view of `language` + `variant` ... is this worth its own helper? (minor)